src/mame/machine/coco.cpp

```diff
@@ -694,7 +694,8 @@ void coco_state::poll_joystick(bool *joyin, uint8_t *buttons)
 			{
 				/* conventional joystick */
 				joyval = analog->input(joystick, joystick_axis);
-				joyin_value = (dac_output() <= (joyval >> 2));
+				float joyval_f = joyval / 10.0;
+				joyin_value = (dac_output() <= joyval_f);
```

Comment:
This doesn’t make much sense – why convert to a `double` (10.0 is a `double` literal), then convert to a `float`, only to compare to a `uint8_t`?

Reply:
My mind was in floating point mode from the edits I had to make to the hi-res portion of this fix. I've simplified this to integer division.